### PR TITLE
doc: Add 0.7 item sizes

### PR DESCRIPTION
**Pull request by ChillerDragon**

### CODE REVIEW

**Comment by ChillerDragon** [comment]
hihi #69

File: doc/snapshot.md [hunk 1]
```diff
@@ -124,3 +124,37 @@ This describes the 0.6 protocol of Teeworlds.
 |       19  |     3  | event_sound_world      |
 |       20  |     3  | event_damage_indicator |
 
+This describes the 0.7 protocol of Teeworlds.
```
Comment (heinrich5991) [inline doc/snapshot.md:127]:
```suggestion
The following describes the 0.7 protocol of Teeworlds.
```
and the same for 0.6 above.

File: doc/snapshot.md [hunk 2]
```diff
@@ -124,3 +124,37 @@ This describes the 0.6 protocol of Teeworlds.
...
+|       21  |     3  | event_sound_world      |
+|       22  |     5  | event_damage           |
+
+Those were added after the first 0.7 release
+and are known but not used to stay backwards compatible
+
+| `type_id` | `size` | name                   |
+| --------: | -----: | ---------------------- |
+|       23  |     1  | obj_player_info_race   |
+|       24  |     3  | obj_game_data_race     |
```
Comment (heinrich5991) [inline doc/snapshot.md:160]: These don't have to be listed here, and I'd exclude them. You could add a sentence like "There were more items added after the initial 0.7 release, but they're not "pre-agreed item sizes" to stay backward compatible."

**Review by heinrich5991 — COMMENTED** [review]
Thanks for adding the new item sizes. I left some comments.

**Comment by heinrich5991** [comment]
Thanks!

**Pull request merged**